pkgs/errorHandler/errorHandler.go

```diff
@@ -1,10 +1,9 @@
-package errorHandler
+package errorhandler
 
 import "log"
```

Comment:
For consistency, maintain the same logger `"github.com/sirupsen/logrus"`. A tip for you, if you want to use the word `log` in your code instead of `logrus` you can alias the import by adding the word `log` before the import. https://www.golangprograms.com/how-to-import-and-use-packages-of-the-same-name.html